lib/activerecord_sqlserver_adapter/jdbc.rb

```diff
@@ -0,0 +1,50 @@
+begin
+  eval 'com.microsoft.sqlserver.jdbc.SQLServerDriver'
+rescue NameError
+  begin
+    require 'sqljdbc4'
+  rescue
+    raise("com.microsoft.sqlserver.jdbc.SQLServerDriver not loaded, try installing the sqljdbc4 gem or put the jdbc driver jar in the java CLASSPATH")
+  end
+end
+
+module ActiveRecordSqlServerAdapter
```

Comment:
We have made a concerted effort to not add more/abstract things to the top level namespace. Could you please change `ActiveRecordSqlServerAdapter::Jdbc` to `ActiveRecord::ConnectionAdapters::SQLServer::Jdbc` namespace with a file in the correct spot at `lib/active_record/connection_adapters/sqlserver/jdbc.rb`. This may seem persnickety, but it is import to use the namespace we setup and file convention.